test/integration/Client/client_lib_metadata_rpc_test.cpp

```diff
@@ -166,6 +170,7 @@ int main() {
         assert(ret == CL_SUCCESS);
         duration_destroy_chronicle += (t2 - t1);
     }
+    client.Disconnect(client_id,flags);
```

Comment:
I agree that it is a good idea to merge connect rpc test with this metadata rpc test. Maybe rename it to just client_lib_rpc_test? I think you can follow what we have in connect rpc test to assert the return value and measure time cost of connect and disconnect.